msg/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ set(msg_files
 	trajectory_waypoint.msg
 	transponder_report.msg
 	tune_control.msg
+	tunnel.msg
```

Comment:
Can we give this a more descriptive name? Maybe something like telemetry_tunnel or mavlink_tunnel?